index.html

```diff
@@ -669,6 +625,35 @@ <h3><dfn>request()</dfn> method</h3>
         rather than having the page to deal with it, e.g by re-requesting the
         lock each time the page becomes visible again.
       </div>
+      <section>
+        <h3><dfn>release()</dfn> method</h3>
+        <p>
+          The <a>release()</a> method, when invoked, MUST run the
+           following steps:
+        </p>
+        <ol class="algorithm">
+          <li>
+            Let |lock| be the <a>context object</a>.
+          </li>
+          <li>
+            Let |record| be the <a>platform wake lock</a>'s
+            <a>state record</a> associated with
+            |lock|'s |type|.
+          </li>
+          <li>
+            Decrease |record|.<a>[[\RequestCounter]]</a> by one.
```

Comment:
This algorithm should also somehow detect whether the parallel steps for the `request()` algorithm are still in progress and if so abort them and cause the associated `Promise` to reject.

Reply:
Fixed, followed the approach used by Payment Request